src/storage/mvcc/txn.rs

```diff
@@ -1655,32 +1606,37 @@ mod tests {
                 write(WriteData::from_modifies(txn.into_modifies()));
                 txn = new_txn(start_ts.into(), cm.clone());
                 txn.extra_op = ExtraOp::ReadOldValue;
-                pessimistic_prewrite(
+                prewrite(
                     &mut txn,
+                    &TransactionProperties {
+                        start_ts: start_ts.into(),
+                        kind: TransactionKind::Pessimistic(TimeStamp::default()),
+                        commit_kind: CommitKind::TwoPc,
+                        primary: b"key",
+                        txn_size: 0,
+                        lock_ttl: 0,
+                        min_commit_ts: TimeStamp::zero(),
+                    },
```

Comment:
It seems we need do this kind of things a lot in tests, maybe we can either:
- Add a `new` or `new_twopc/new_1pc/new_async_commt` for TransactionProperties, or
- Use a `TransactionPropertiesBuilder` to construct it, and fill some fileds with default value.